admin/src/board/commons.ts

```diff
@@ -0,0 +1,48 @@
+const GameState = {
+  placeTile: 0,
+  movePawn: 1,
+  completed: 2,
+};
+
+export const renderGameState = ({ game_state }: { game_state: number }) => {
+  switch (game_state) {
```

Comment:
Can you extract the case and return to an object? You'll reduce the complexityn get rid of the switch statement, and surface an important piece of data.